### DISCUSSION TITLE
Enable to send multi SELECT and get multi results

### DISCUSSION THREAD

**Discussion by RickTrder** [dbeaver/dbeaver#34626] [Q&A] [answered]
Enable to send multi SELECT and get multi results  (at least 3 SELECT)

Highlight 3 SELECT with mouse and clicking "Run" button.  

 Using SnowFlake JDBC connectors.

Enable to the 3 results (back to back) in the same panel.

**Comment by E1izabeth** [comment, marked as answer]
If I correctly understood you, we have such mode for execution as a script with this button ![image](https://github.com/dbeaver/dbeaver/assets/28875055/be3d3a7f-34ab-4c8c-b48f-d03effa93314)

You can enable it in the context menu
![image](https://github.com/dbeaver/dbeaver/assets/28875055/a3b68ed2-bb39-478b-80d0-d110e5108b0d)

**Comment by RickTrder** [comment]
Ok thanks!
